dev-tools/cloud/launch_cloud_e2e_tests.sh

```diff
@@ -7,9 +7,12 @@ CLOUD_TESTING_BASE="$(dirname $0)"
 cleanup() {
   r=$?
 
-  echo "--- Cleaning deployment"
-  make -C "${CLOUD_TESTING_BASE}" cloud-clean
-
+  if [ -f ${CLOUD_TESTING_BASE}/terraform/.terraform.lock.hcl ] ; then
+    echo "--- Cleaning deployment"
+    make -C "${CLOUD_TESTING_BASE}" cloud-clean
+  else
+      echo "+++ Skipped cleaning deployment, no terraform files"
```

Comment:
Just taking a look to the build , probably it's better to just show the echo wihtout any Buildkite formatting 

```suggestion
      echo "Skipped cleaning deployment, no Terraform files"
```